### ISSUE TITLE
bug(YouTube): Patched app crashes after first launch or clearing data

### ISSUE THREAD

**Issue by WsNyc00** [inotia00/ReVanced_Extended#2582]
### Type

Other

### Tools used

RVX Manager

### Application

Youtube: 19.44.39
RVX: 19.44.39
RVX patches: v5.1.1

### Bug description

After installing and launching app for the first time or clearing data, patched app crashes.
Most likely its crashes only after clearing app data. (Sometimes crashes, sometimes not)
Not always issue reproduces after launches app for first time or clearing data.

### Error logs
```shell

```

### Solution

App only crashes on first launch, second and other launches are fine.

### Additional context

Used default patches, only added "visual preferences icons" in the options, deleted all translations, except my native language.
And also tested with all default selection, same problem, i know its small problem, but previously patched app didnt had this problem.
EDIT: On my phone which has Android 14 Only crashing for first time after clearing data and still can't always replicate same crash, sometimes crashes, sometimes not.

### Device Environment

Android 8.1.0 (Tablet)
Android 14 / OneUI 6.1

### Acknowledgements

- [X] This issue does not reproduce on unpatched YouTube or YT Music.
- [X] This issue is not a duplicate of an existing bug report.
- [X] I did not use any settings marked as `Experimental Flags`.
- [X] I did not use `pre-release` (dev branch).
- [X] I have patched the APK according to the [documentation](https://github.com/inotia00/revanced-documentation#readme).
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.
- [X] I have written the title and contents in English.

**Comment by inotia00** [comment]
It's weird, the issue is not reproduced on my side

It may be an issue that can only be reproduced in very specific circumstances

**Comment by WsNyc00** [comment]
> It's weird, the issue is not reproduced on my side
> 
> It may be an issue that can only be reproduced in very specific circumstances
Like i said, its not always crashes, sometimes doesn't crash at all after clearing data, or crashes, once, but launching app after crash its doesnt crash anymore. Try quickly launch app after clearing data and try do multiple times.

**Comment by itsmartashub** [comment]
Hi there, same crash on the first launch for the `A13`, **rooted**, `HyperOS 1.0.6.0` Xiaomi.eu custom ROM.
Really hope there is something usefull in next log file:

### Log file of first load crash:
[rvx-yt-v19.44.39-foxlog.txt](https://github.com/user-attachments/files/18219147/rvx-yt-v19.44.39-foxlog.txt)

### Patched selection: 
[selected_patches_2024-12-21_17_05_32.json](https://github.com/user-attachments/files/18219161/selected_patches_2024-12-21_17_05_32.json)

PS. before every update I unmount, then `Clear All Data` and `Force Stop`, then uninstall it and deep clean all with SD Maid app. So it should be pretty clean patching and installing...

**Comment by kyan5221** [comment]
on my Android 12 device, the app crashes when I run it for the first time after deleting app data. but it doesn't crash on Android 14 devices.

**Comment by inotia00** [comment]
reflected in [revanced-patches-5.1.2](https://github.com/inotia00/revanced-patches/releases/tag/v5.1.2)

**Comment by WsNyc00** [comment]
I don't know if i should open new issue or write here,
patched app still crashes for first time after installing or after clearing data, crashes in both 5.1.2 and 5.1.3

**Comment by inotia00** [comment]
The issue has been closed because the app crash included in the provided logcat has been resolved

> I don't know if i should open new issue or write here, patched app still crashes for first time after installing or after clearing data, crashes in both 5.1.2 and 5.1.3

If so, please open a new issue and attach the logcat
